src/components/Event/UsersEvents/index.tsx

```diff
@@ -52,12 +53,18 @@ const UsersEvents = observer((props: Props) => {
 
     return (
         <Tabs lazy>
-            <TabItem value='my-events' label='Unveröffentlicht'>
+            <TabItem value='my-events' label={translate({
+                message: 'Not published',
+                id: 'components.event.usersevents.index.tabitem.notpublished',
+                description: 'Text not published'})}>
```

Comment:
Style Convention for readability:
When a tag has an attribute, that does not fit on one line, every attribute is started on a new line:
```suggestion
            <TabItem
                value='my-events' 
                label={translate({
                    message: 'Not published',
                    id: 'components.event.usersevents.index.tabitem.notpublished',
                    description: 'Text not published'
                })}
            >
```